templates/build-pod/template.json

```diff
@@ -0,0 +1,146 @@
+{
```

Comment:
We're suddenly switching from yaml to json here. I would stick with one, which for the most part, yaml seems to be the preferred format.